server/models/Permission.js

```diff
@@ -45,4 +45,11 @@ permissionSchema.index(
   }
 );
 
+permissionSchema.index(
```

Comment:
these will probably need to change to include the fullText index, i guess it's work to be done after this PR merges?

Reply:
Agreed.